### PR TITLE
docs: installing on macOS in a restricted environment

### LINKED ISSUE #1034: Docs: Installation in an AirGapped Environment
### Is your feature request related to a problem? Please describe

Following the work on #185 we would like to provide a dedicated guide for installation in an AirGapped Environment.

### Describe the solution you'd like

Guide in the installation section that explain how to manage installation of Podman Desktop in an AirGapped environment

- [x] on Linux https://github.com/containers/podman-desktop/pull/1118 https://podman-desktop.io/docs/Installation/linux-install#archive
- [x] on macOS https://github.com/containers/podman-desktop/pull/1493
- [x] on Windows https://github.com/containers/podman-desktop/pull/1422

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by themr0c**
* added assets to the downloads page
* fixes #1034

### CODE REVIEW

**Review by benoitf — DISMISSED** [review]
checking on https://63ef51b5caa1af586152d0b6--podman-desktop-pr.netlify.app/downloads

I've for macOS binary: windowsSetupAsset is not defined (so it's still strange to name something Windows when on macOS)

also the zip should not be listed as a downloadable archive. Only the dmg should be listed

File: website/src/pages/downloads/macOS.tsx [hunk 1]
```diff
@@ -93,6 +119,18 @@ export function MacOSDownloads(): JSX.Element {
...
+              to={downloadData.airgapsetup}>
+              <FontAwesomeIcon size="1x" icon={faDownload} className="mr-2" />
+              macOS universal installer for restricted environments
+            </Link>
+            <Link
+              className="underline inline-flex dark:text-white text-purple-600 hover:text-purple-300 py-2 px-6 text-md font-semibold"
+              to={downloadData.airgaparchive}>
+              <FontAwesomeIcon size="1x" icon={faDownload} className="mr-2" />
+              macOS universal archive for restricted environments
+            </Link>
```
Comment (themr0c) [inline website/src/pages/downloads/macOS.tsx]:
```suggestion
```

File: website/src/pages/downloads/macOS.tsx [hunk 2]
```diff
@@ -42,6 +66,8 @@ async function grabfilenameforMac(
     universal: unifiedMacLinj.browser_download_url,
     x64: intelLink.browser_download_url,
     arm64: armLink.browser_download_url,
+    airgapsetup: universalMacAirgapDmgAsset.browser_download_url,
+    airgaparchive: universalMacAirgapZipAsset.browser_download_url,
```
Comment (themr0c) [inline website/src/pages/downloads/macOS.tsx]:
```suggestion
```

File: website/src/pages/downloads/macOS.tsx [hunk 3]
```diff
@@ -26,6 +26,30 @@ async function grabfilenameforMac(
...
+    asset =>
+      (asset.name as string).endsWith('universal.zip') &&
+      asset.name.includes('airgap') &&
+      asset.name !== windowsSetupAsset.name,
+  );
+  if (universalMacAirgapZipAssets.length !== 1) {
+    throw new Error('Unable to find macOS universal archive for restricted environments');
+  }
+  const universalMacAirgapZipAsset = universalMacAirgapZipAssets[0];
+
```
Comment (themr0c) [inline website/src/pages/downloads/macOS.tsx]:
```suggestion
```

File: website/src/pages/downloads/macOS.tsx [hunk 4]
```diff
@@ -26,6 +26,18 @@ async function grabfilenameforMac(
   }
   const intelLink = intelMacDmg[0];
 
+  /* Find macOS universal installer for restricted environments */
+  const universalMacAirgapDmgAssets = assets.filter(
+    asset =>
+      (asset.name as string).endsWith('universal.dmg') &&
+      asset.name.includes('airgap') &&
+      asset.name !== windowsSetupAsset.name,
```
Comment (themr0c) [inline website/src/pages/downloads/macOS.tsx]:
```suggestion
```

File: website/src/pages/downloads/macOS.tsx [hunk 5]
```diff
@@ -26,6 +26,18 @@ async function grabfilenameforMac(
   }
   const intelLink = intelMacDmg[0];
 
+  /* Find macOS universal installer for restricted environments */
+  const universalMacAirgapDmgAssets = assets.filter(
+    asset =>
+      (asset.name as string).endsWith('universal.dmg') &&
+      asset.name.includes('airgap') &&
```
Comment (themr0c) [inline website/src/pages/downloads/macOS.tsx]:
```suggestion
      asset.name.includes('airgap')
```

**Comment by benoitf** [comment]
image


I would try to reduce the width of the text, as we're also inside the macOS box I would not repeat macOS and the DMG is not an installer

maybe something like: `DMG for restricted environments` (or Disk Image) or...

File: website/src/pages/downloads/macOS.tsx [hunk 6]
```diff
@@ -26,6 +26,17 @@ async function grabfilenameforMac(
...
   const intelLink = intelMacDmg[0];
 
+  /* Find macOS universal installer for restricted environments */
+  const universalMacAirgapDmgAssets = assets.filter(
+    asset =>
+      (asset.name as string).endsWith('universal.dmg') &&
+      asset.name.includes('airgap')
+  );
+  if (universalMacAirgapDmgAssets.length !== 1) {
+    throw new Error('Unable to find macOS universal installer for restricted environments');
```
Comment (themr0c) [inline website/src/pages/downloads/macOS.tsx]:
```suggestion
    throw new Error('Unable to find macOS DMG for restricted environments');
```

File: website/src/pages/downloads/macOS.tsx [hunk 7]
```diff
@@ -26,6 +26,17 @@ async function grabfilenameforMac(
   }
   const intelLink = intelMacDmg[0];
 
+  /* Find macOS universal installer for restricted environments */
+  const universalMacAirgapDmgAssets = assets.filter(
+    asset =>
+      (asset.name as string).endsWith('universal.dmg') &&
+      asset.name.includes('airgap')
```
Comment (themr0c) [inline website/src/pages/downloads/macOS.tsx]:
```suggestion
      asset.name.includes('airgap'),
```

File: website/src/pages/downloads/macOS.tsx [hunk 8]
```diff
@@ -93,6 +105,12 @@ export function MacOSDownloads(): JSX.Element {
               <FontAwesomeIcon size="1x" icon={faDownload} className="mr-2" />
               Arm
             </Link>
+            <Link
+              className="underline inline-flex dark:text-white text-purple-600 hover:text-purple-300 py-2 px-6 text-md font-semibold"
+              to={downloadData.airgapsetup}>
+              <FontAwesomeIcon size="1x" icon={faDownload} className="mr-2" />
+              macOS universal installer for restricted environments
```
Comment (themr0c) [inline website/src/pages/downloads/macOS.tsx]:
```suggestion
              universal DMG for restricted environments
```

File: website/src/pages/downloads/macOS.tsx [hunk 9]
```diff
@@ -26,6 +26,17 @@ async function grabfilenameforMac(
   }
   const intelLink = intelMacDmg[0];
 
+  /* Find macOS universal installer for restricted environments */
```
Comment (themr0c) [inline website/src/pages/downloads/macOS.tsx]:
```suggestion
  /* Find macOS universal DMG for restricted environments */
```

File: website/src/pages/downloads/macOS.tsx [hunk 10]
```diff
@@ -26,6 +26,17 @@ async function grabfilenameforMac(
   }
   const intelLink = intelMacDmg[0];
 
+  /* Find macOS universal DMG for restricted environments */
```
Comment (themr0c) [inline website/src/pages/downloads/macOS.tsx]:
```suggestion
  /* Find macOS universal Disk Image for restricted environments */
```

File: website/src/pages/downloads/macOS.tsx [hunk 11]
```diff
@@ -26,6 +26,17 @@ async function grabfilenameforMac(
...
+  const universalMacAirgapDmgAssets = assets.filter(
+    asset =>
+      (asset.name as string).endsWith('universal.dmg') &&
+      asset.name.includes('airgap'),
+  );
+  if (universalMacAirgapDmgAssets.length !== 1) {
+    throw new Error('Unable to find macOS DMG for restricted environments');
```
Comment (themr0c) [inline website/src/pages/downloads/macOS.tsx]:
```suggestion
    throw new Error('Unable to find Apple Disk Image for restricted environments');
```

File: website/src/pages/downloads/macOS.tsx [hunk 12]
```diff
@@ -93,6 +105,12 @@ export function MacOSDownloads(): JSX.Element {
               <FontAwesomeIcon size="1x" icon={faDownload} className="mr-2" />
               Arm
             </Link>
+            <Link
+              className="underline inline-flex dark:text-white text-purple-600 hover:text-purple-300 py-2 px-6 text-md font-semibold"
+              to={downloadData.airgapsetup}>
+              <FontAwesomeIcon size="1x" icon={faDownload} className="mr-2" />
+              universal DMG for restricted environments
```
Comment (themr0c) [inline website/src/pages/downloads/macOS.tsx]:
```suggestion
              Disk Image for restricted environments
```

**Comment by cdrage** [comment]
@themr0c can you rebase against main / repush so we can trigger a netlify build / test it out?

**Review by benoitf — APPROVED** [review]
checked the link on netlify
it works nicely 👍

**Pull request merged**